recipes/example/meta.yaml

```diff
@@ -9,7 +11,7 @@ source:
     fn: simplejson-{{ version }}.tar.gz
     url: https://pypi.io/packages/source/s/simplejson/simplejson-{{ version }}.tar.gz
     # md5 is OK but sha256 is desirable.
-    sha256: d58439c548433adcda98e695be53e526ba940a4b9c44fb9a05d92cd495cdd47f
+    {{ hash_type }}: {{ hash_val }}
```

Comment:
Yeah, it seems static enough. Though I haven't argued with anyone that went that route.